Retrovizor.Domain/Repositories/Implementations/VehicleRepository.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Retrovizor.Data.Entities;
+using Retrovizor.Data.Entities.Models;
+using Retrovizor.Domain.Repositories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Retrovizor.Domain.Repositories.Implementations
+{
+    public class VehicleRepository : IVehicleRepository
+    {
+        VehicleRepository(RetrovizorContext context)
+        {
+            _context = context;
+        }
+        private readonly RetrovizorContext _context;
+
+        public List<Vehicle> GetAllVehicles()
```

Comment:
mislin da bi bilo korisnije ubacit vehicles by driving school ili by instructor (ovo ce nan bit obavezan feature al to mozemo kasnije ubacit)